pkg/azureclients/containerserviceclient/azure_containerserviceclient.go

```diff
@@ -96,7 +96,7 @@ func (c *Client) Get(ctx context.Context, resourceGroupName string, managedClust
 	}
 
 	result, rerr := c.getManagedCluster(ctx, resourceGroupName, managedClusterName)
-	_ = mc.Observe(rerr.Error())
```

Comment:
This method used to return a error, but it's never used. Switched that to get rid of the return type.